Stratis.Bitcoin.Features.BlockStore.Tests/Stratis.Bitcoin.Features.BlockStore.Tests.csproj

```diff
@@ -27,9 +27,9 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="DBreeze" Version="1.85.0" />
+    <PackageReference Include="DBreeze" Version="1.89.0" />
```

Comment:
in theory, why not, but have you checked that it works properly?

Reply:
Yes, tests pass properly.